server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/message/view/MessageFullViewFactoryTest.java

```diff
@@ -779,7 +783,7 @@ void previewBodyShouldReturnStringEmptyWhenNoMeaningHtmlBodyAndNoTextBody() thro
 
         assertThat(testee)
             .extracting(MessageFullView::getPreview, MessageFullView::getHtmlBody, MessageFullView::getTextBody)
-            .containsExactly(MessagePreviewGenerator.NO_BODY, Optional.of("<html><body></body></html>"), Optional.empty());
+            .containsExactly(PreviewDTO.of("(Empty)"), Optional.of("<html><body></body></html>"), Optional.empty());
```

Comment:
`PreviewDTO.of("(Empty)")` should be replaced by `PreviewDTO.of("")` everywhere we can in unit tests to ensure fixing the local bug is as easy as possible.

Reply:
I don't see any possible test I can use `PreviewDTO.of("")`

Reply:
Here for instance?